src/pyxtp/CMakeLists.txt

```diff
@@ -0,0 +1,37 @@
+find_package(Python 3 REQUIRED COMPONENTS Interpreter Development)
+find_package(pybind11 REQUIRED)
+
+file(GLOB_RECURSE BIND_SOURCES binds/*.cc)
+
+pybind11_add_module(xtp_binds ${BIND_SOURCES})
+target_include_directories(xtp_binds
+ PUBLIC
+ binds/include
+)
+
+target_compile_options(xtp_binds
+ PUBLIC
+)
+set_target_properties(xtp_binds
+  PROPERTIES
+  CXX_STANDARD 14
+  CXX_STANDARD_REQUIRED ON
+)
+
+
+target_link_libraries(xtp_binds
+ PUBLIC
+ VOTCA::votca_xtp
+)
+
+# # DPYXTP_VERSION_INFO is defined in the setup.py file
+target_compile_definitions(xtp_binds PRIVATE VERSION_INFO=${XTP_BINDS_VERSION_INFO})
+
+if(NOT PYTHON_INSTDIR)
+  execute_process(
+    COMMAND
+      ${Python_EXECUTABLE} -c
+      "import distutils.sysconfig as cg; print(cg.get_python_lib(prefix='${CMAKE_INSTALL_PREFIX}', plat_specific=True, standard_lib=False))"
+    OUTPUT_VARIABLE PYTHON_INSTDIR OUTPUT_STRIP_TRAILING_WHITESPACE)
+endif(NOT PYTHON_INSTDIR)
+install(TARGETS xtp_binds LIBRARY DESTINATION ${PYTHON_INSTDIR})
```

Comment:
Do we need to install all `*.py` as well?

Reply:
Not just the C++ and the generated library. I still don't know how to load the library with Python but the `setup.py` invokes cmake and then link it. I am trying to understand how it works

Reply:
Check my latest commit! After the install, I did:
```
source VOTCARC.bash
export PYTHONPATH=/path/to/site-packages
python -c "from pyxtp import xtp_binds"
```

so the install seems to work.

Reply:
see https://github.com/votca/tools/pull/340